### ISSUE TITLE
Validator does not report missing IDs in the consecutive rows.

### ISSUE THREAD

**Issue by elsylambert** [unfoldingWord/tc-create-app#812]
There are missing IDs in TIT 1:1 line 8, 9 and 10.
![Screen Shot 2021-04-13 at 5.26.03 PM.png](https://images.zenhubusercontent.com/5d02efb2f425650e637abe10/2b8ce812-5616-435e-87fc-dd7e4272ef2f)
Validator reports Missing ID for line 8, but not for line 9 and line 10 which is an empty row.
![Screen Shot 2021-04-13 at 5.23.30 PM.png](https://images.zenhubusercontent.com/5d02efb2f425650e637abe10/762e7182-6bac-466d-88dc-e6c5f01a4a4e)

**Comment by birchamp** [comment]
@RobH123 Please note this for the validator.

**Comment by RobH123** [comment]
Yes, there's definitely some funny stuff going on here. There is an error for Line 10, but it's the r2gl row! I suspect that the data being sent to the CV functions is NOT what's being displayed in the top screenshot.
